src/Components/Web/src/Forms/InputTextArea.cs

```diff
@@ -29,6 +30,7 @@ public class InputTextArea : InputBase<string?>
         [DisallowNull] public ElementReference? Element { get; protected set; }
 
         /// <inheritdoc />
+        [DynamicDependency(BlazorComponent, typeof(InputTextArea))]
```

Comment:
Is it an attempt to workaround a linker solution, or do you want to keep these long-term?

Reply:
This is a workaround and hopefully something that should be resolved as part of https://github.com/mono/linker/issues/1806. Unfortunately the annotation does not particularly help because of how these APIs get used. I switched back to the linker descriptors for the time being